plugins/com.google.cloud.tools.eclipse.appengine.libraries/src/com/google/cloud/tools/eclipse/appengine/libraries/ILibraryClasspathContainerResolverService.java

```diff
@@ -30,8 +30,27 @@
  */
 public interface ILibraryClasspathContainerResolverService {
 
+  // extract to a standalone enum?
```

Comment:
I was unsure whether we should have this as an `AppEngineStandardRuntime` enum or keep this broader `AppEngineRuntime`.  But I think we could do something like:

```java
enum AppEngineRuntime {
   STANDARD_JAVA_7, STANDARD_JAVA_8, FLEX_JAR, FLEX_WAR, FLEX_CUSTOM;  // definitions skipped

   static EnumSet<AppEngineRuntime> standardRuntimes() {
      return EnumSet.of(STANDARD_JAVA_7, STANDARD_JAVA_8);
   }

   static EnumSet<AppEngineRuntime> flexbileRuntimes() {
      return EnumSet.of(FLEX_JAR, FLEX_WAR, FLEX_CUSTOM);
   }

   boolean isStandard() { return standardRuntimes().contains(this); }
   boolean isFlexible() { return flexibleRuntimes().contains(this); }
}
```

We could consider adding `isGA()` type methods too, to pop up a warning if the user selects a non-GA version.